### ISSUE TITLE
Build status: dockerfile, what does that mean?

### ISSUE THREAD

**Issue by AlexisMontagne** [docker/hub-feedback#281]
Two of my automatic builds have the status `dockerfile`. 

![screen shot 2015-08-14 at 13 16 27](https://cloud.githubusercontent.com/assets/1418979/9283229/e16ea8fc-4286-11e5-861e-912d8c6b4d42.png)

**Comment by chambm** [comment]
:+1: See this myself. No idea what it means. Is it dockerspeak for "in progress"?

**Comment by tjdett** [comment]
:+1: It appears Docker Hub updates the Dockerfile, but that really doesn't help define where the build is in the cycle.

**Comment by ChristopherBiscardi** [comment]
There are a series of ~10 statuses a build can be in aside from error. (I thought it was in the docs but I can't find it right now). We are working on ensuring the meanings of these statuses are easily discoverable but it is not our top priority right now since they all essentially mean "in-progress".

**Comment by djsly** [comment]
:+1: Same here, I was waiting for a triggered build and saw it stuck with a status: dockerfile, I issued a second triggered and they are now both stuck at dockerfile.

It should be good to know that they are indeed in progress. 

Maybe have a progress bar, expected available time ?

**Comment by bluszcz** [comment]
Experiencing same issue here:https://hub.docker.com/r/bluszcz/plone5/builds/ also getting unexpected errors...

**Comment by paynerc** [comment]
:+1:

**Comment by robertoandrade** [comment]
more like :-1: :)

**Comment by chambm** [comment]
Nah, the status is: "thumb"

**Comment by cpttripzz** [comment]
status seems more like:
....................../´¯/) 
....................,/¯../ 
.................../..../ 
............./´¯/'...'/´¯¯`·¸ 
........../'/.../..../......./¨¯\ 
........('(...´...´.... ¯~/'...') 
.........\.................'...../ 
..........''...\.......... _.·´ 
............\..............( 
..............\.............\...

**Comment by dciccale** [comment]
i have the same question.

the statuses that i have seen are:

pending (which i understand of building...)
error
done
dockerfile (is this the limbo?)

**Comment by shadowbq** [comment]
:8ball: Tell me my status..

**Comment by dciccale** [comment]
what i found out, is that if you wait some minutes, the status actually changes to done or error. 

i think dockerfile means "executing dockerfile instructions"

**Comment by oryband** [comment]
:+1: 
the whole ui needs many improvements. specifically the whole image status page is very confusing and ugly.

**Comment by JikkuJose** [comment]
@oryband +1, the site really needs a redo.

**Comment by staff0rd** [comment]
:+1:

**Comment by raphi** [comment]
:+1:  I've seen "claimed" too, no idea either.

**Comment by ChristopherBiscardi** [comment]
An update to these statuses is coming soon.

**Comment by oryband** [comment]
maybe at least some documentation on the current statuses? keeping us in the dark here

**Comment by oryband** [comment]
from what i understand:
- `pending`: means the build is waiting for an available dockerhub worker?
- `claimed`: means a worker has *started* the build?
- `dockerfile`: means the dockerfile is being read?
- `done`: means the build has been built and pushed to the repo?

**Comment by ChristopherBiscardi** [comment]
@oryband The states are being changed to

* success
* error
* building
* queued

These were chosen to be immediately obvious to a new user (without explanations such as the ones `dockerfile` needs), so please let me know if any of the states are unclear.

**Comment by dciccale** [comment]
@ChristopherBiscardi those are self explanatory thanks

**Comment by oryband** [comment]
@ChristopherBiscardi yes, they're fine. *However*, you should still document this somewhere, and maybe put a little (?) button or some hovering tooltip. Relying on people's rational logic never works 100%.

**Comment by mghazizadeh** [comment]
https://github.com/docker/docs.docker.com/issues/82
